communitiesuk/funding-service-design-frontend
# Fs 2432 start page

https://digital.dclg.gov.uk/jira/browse/FS-2432
https://digital.dclg.gov.uk/jira/browse/FS-2183
https://app.mural.co/t/mhclg2837/m/mhclg2837/1675177182644/578e9019ed4a6da033b5de2183289ca7298e5f12?sender=u94fce4b3d87aef5380039386

### Change description
These changes are to make the fund start page work for any fund, based on URL path. The existing index page (`/`) now redirects to `/cof/r2w3` as COF R2W3 is still live and uses that as it's start page. However all funds that exist in the fund store will now be accessible at `/fund_short_name/round_short_name` eg. `/cof/r2w2` or `/COF/R2W3`.

If you try and access this using just a fund short code (eg. `/cof/`) you will get redirected to the most recent round in that fund.

Added a new template `fund_start_page.html` which parameterises all the text on the start page and these values are now pulled from the fund store.



### How to test
Run locally with the docker runner. Navigate to the following URLs
- http://localhost:3008/ - should render as currently
- http://localhost:3008/cof/r2w3 - should render as above
- http://localhost:3008/cof/r2w2 - should display Round 2 Window 2 as closed
- http://localhost:3008/cof - redirects to r2w3 page above (As that's the most recently open window)
- http://localhost:3008/made_up_fund - 404


### Screenshots of UI changes (if applicable)
No visible changes to users
